src/janitor.c

```diff
@@ -550,6 +551,11 @@ static void pool_server_maint(PgPool *pool)
 	usec_t now = get_cached_time();
 	PgSocket *server;
 
+	usec_t effective_query_timeout;
+	usec_t effective_idle_transaction_timeout;
+	usec_t user_query_timeout;
+	usec_t user_idle_transaction_timeout;
+
```

Comment:
nit: let's move this inside the loop. No need to declare these variables with this large of a scope.

Reply:
Moved these variable declarations into the loop with the latest commit.